### ISSUE TITLE
Supporting Rapido UHF / Rapido 2?

### ISSUE THREAD

**Issue by dflemstr** [sorted01/Filametrix#27]
I'm wondering if anyone has adapted to the new Rapido 2 series, and/or the UHF version with extended air channels?  Otherwise I might hack something together

**Comment by bartoszwas** [comment]
Mine Rapido 2 fits in the old Rapido 1 stl. I’m not sure does every Rapido 2 will fit but mine does…

**Comment by dflemstr** [comment]
Makes sense; however in any case the UHF version is very different and would require a special part for sure.

**Comment by d0m1n1qu3** [comment]
but there are two versions in the voron git one for rapido 1 and one for rapido 2 .. there must be a difference

**Comment by dflemstr** [comment]
Cool, in turn I hacked together a Rapido 2 UHF version with longer cooling ducts (the back part is the same, only the front part changed)
[fillametrix-rapido-2-uhf.zip](https://github.com/sorted01/Filametrix/files/13651783/fillametrix-rapido-2-uhf.zip)
This is a Blender hack of the model you posted above, because I couldn't find a manifold version of the original parts anywhere that would let me do the mod in OpenSCAD

**Comment by BrewNinja** [comment]
Did you test your version @dflemstr  ? Im looking to do the same thing.

**Comment by dflemstr** [comment]
@BrewNinja yes, I am currently using my version in a printer!

**Comment by d0m1n1qu3** [comment]
and here my scad with fixed stls .. is also doing his job obn my brand new voron v2.4 :-D  but only HF not UHF
[SBRapidoV2FilametrixHF.zip](https://github.com/sorted01/Filametrix/files/13801229/SBRapidoV2FilametrixHF.zip)

**Comment by gordon-quad** [comment]
Can we also have Rapido v1 UHF version please?

**Comment by dflemstr** [comment]
I don't think we should share the hacked versions in the repo. It is better to use the original CAD to create a properly designed version in my opinion!

**Comment by JarTheHead** [comment]
ah mb i was under the assumption the second one was done with CAD

**Comment by dflemstr** [comment]
Well, technically, SCAD should count as CAD, of course, but I was hoping for a `.step` based version (or Fusion 360/whatever was used to produce the `.step` files) to retain the parametric curves, etc., from the original model.
